index.d.ts

```diff
@@ -34,7 +34,7 @@ declare const loadJsonFile: {
 	})();
 	```
 	*/
-	<T = JsonValue>(filePath: string, options?: loadJsonFile.Options): Promise<T>;
+	<ValueType = JsonValue>(filePath: string, options?: loadJsonFile.Options): Promise<ValueType>;
```

Comment:
If you're going to be very clear, it could be named `ReturnValueType`.

Reply:
Done!